projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import abc
+from dataclasses import dataclass
+
+from vdk.plugin.meta_jobs.api import meta_job
+
+
+class IDataJobExecutor(abc.ABC):
```

Comment:
Maybe a part of the extensible API

Reply:
in fact, I am looking at core's IStandaloneDataJob that has some common properties used by this plugin domain model, for example name, args (and team that is matching the outer job I guess);

Reply:
It's an implementation internal interface. It's not meant to be an API (it's in 'vdk.api' package)